### PR TITLE
CI - Release Automation

### PR DESCRIPTION
**Pull request by CrazyOptimist**
#### Summary
- Use the builder image provided by electron userland
- Add job for packaging linux installer

#### Note
- Only linux(ubuntu/debian) build is created for now, Windows and Mac will be added in the next PR.
- Branch filter(master and release only) will be added after this PR would be approved.

Edit: Filters have been added (for only master for now).

### CODE REVIEW

**Review by Dicklesworthstone — APPROVED** [review]

**Review by ngvtuan — APPROVED** [review]
Look fine to me but: 
- I can't access the CI to check the result.
- Please merge with the latest code of the master branch

**Review by ngvtuan — DISMISSED** [review]
Look fine to me but:

- I can't access the CI to check the result.
- Please merge with the latest code of the master branch

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,11 +35,40 @@ jobs:
...
     jobs:
       - install-deps
       - lint-and-test:
           requires:
             - install-deps
+      - build-ubuntu:
+          filters: &filters-build
+            branches:
+              only:
+                - master
```
Comment (br4in3x) [inline .circleci/config.yml:72]: We should be building only for `release/*` branches. Also, please consider adding an automatic artifact upload to a draft release (perhaps in another PR, seems to be complex enough)

Comment (CrazyOptimist) [reply .circleci/config.yml:72]: ok

**Review by ngvtuan — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (CrazyOptimist) [reply .circleci/config.yml]: https://github.com/pastelnetwork/pastel-electron-wallet/pull/107/commits/26f72d4701afd75eac773991079a2159047808ad

**Review by Dicklesworthstone — APPROVED** [review]

**Pull request merged**